controllers/reconcilers/configuration/token_refresher.go

```diff
@@ -235,7 +235,7 @@ func (r *Reconciler) reconcileTokenRefresher(ctx context.Context, cr *v1.Observa
 		}
 
 		promtailDisabled := false
-		if index.Config.Promtail == nil || index.Config.Promtail.Enabled == false {
+		if index.Config.Promtail == nil || !index.Config.Promtail.Enabled {
```

Comment:
any particular reason for this?

Reply:
@pb82 No particular reason except for consistency in the code